src/connector/src/parser/additional_columns.rs

```diff
@@ -55,6 +56,18 @@ pub static COMPATIBLE_ADDITIONAL_COLUMNS: LazyLock<HashMap<&'static str, HashSet
             (OPENDAL_S3_CONNECTOR, HashSet::from(["file", "offset"])),
             (S3_CONNECTOR, HashSet::from(["file", "offset"])),
             (GCS_CONNECTOR, HashSet::from(["file", "offset"])),
+            (
+                MYSQL_CDC_CONNECTOR,
+                HashSet::from(["timestamp", "partition", "offset"]),
+            ),
+            (
+                POSTGRES_CDC_CONNECTOR,
+                HashSet::from(["timestamp", "partition", "offset"]),
+            ),
+            (
+                MONGODB_CDC_CONNECTOR,
+                HashSet::from(["timestamp", "partition", "offset"]),
+            ),
```

Comment:
no need for cdc connectors to expose partition and offset?

Reply:
This part enforce the `"partition", "offset"` must exist, I am not sure the intention. cc @xxchan 
https://github.com/risingwavelabs/risingwave/blob/18a05240bf6fb6741cf0b14fda4acf6fd801a829/src/connector/src/parser/additional_columns.rs#L205-L231

Reply:
Well, this is introduced in https://github.com/risingwavelabs/risingwave/pull/14524/files#diff-6dec7cb267e3578f43aae619c514713a625f02b8a66fe903b7cd8c245e9e6b08.

Reply:
IIRC, the cdc stream job does not share the same plan with the mq jobs. The additional column requirement does not apply to the CDC ones.

Reply:
fixed. separated additional columns of cdc backfill table to another path.